pin/gc/gc.go

```diff
@@ -95,10 +101,13 @@ func GC(ctx context.Context, bs bstore.GCBlockstore, dstor dstore.Datastore, pn
 					if err != nil {
 						errors = true
 						output <- Result{Error: &CannotDeleteBlockError{k, err}}
-						//log.Errorf("Error removing key from blockstore: %s", err)
+						// log.Errorf("Error removing key from blockstore: %s", err)
 						// continue as error is non-fatal
 						continue loop
 					}
+					if err := cleanup.Cleanup(k); err != nil {
```

Comment:
**NOTE** I'm in the process of removing this from the GC path and making clean up a part of reproviding.